src/compute/src/compute_observer/observer_manager.rs

```diff
@@ -83,9 +106,13 @@ impl ObserverNodeImpl for ComputeObserverNode {
 }
 
 impl ComputeObserverNode {
-    pub fn new(filter_key_extractor_manager: FilterKeyExtractorManagerRef) -> Self {
+    pub fn new(
+        filter_key_extractor_manager: FilterKeyExtractorManagerRef,
+        local_version_manager: Weak<LocalVersionManager>,
```

Comment:
I think Weak ref is unnecessary, because after ObserverManager::start is joined, the whole instance is gone?

Reply:
> I think Weak ref is unnecessary(Arc is fine), because after ObserverManager::start is joined, the whole instance is gone?

just in case

Reply:
> I think Weak ref is unnecessary(Arc is fine), because after ObserverManager::start is joined, the whole instance is gone?

fixed